website/docs/d/container_registry.markdown

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
could we also add this to the sidebar? here's an example of a Data Source in the sidebar: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L62-L64